src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json

```diff
@@ -123,6 +123,33 @@
     }
   },
   "packs": {
+    "Microsoft.Maui.Core.Runtime.win": {
+      "kind": "framework",
+      "version": "@VERSION@",
+      "alias-to": {
+        "win-x86": "Microsoft.Maui.Core.Runtime.win-x86",
+        "win-x64": "Microsoft.Maui.Core.Runtime.win-x64",
+        "win-arm64": "Microsoft.Maui.Core.Runtime.win-arm64"
+      }
```

Comment:
One nice benefit of this alias, is that `dotnet workload install maui` will only install the pack that matches your OS. The rest would come from NuGet restore if needed (like building `-r win-arm64` on a x64 PC).